.github/workflows/update-snapshot.yml

```diff
@@ -4,118 +4,122 @@ on:
   # Run any time when a new label added to the PR
   pull_request:
     types: [opened, synchronize, reopened, labeled]
+  workflow_dispatch:
 
 jobs:
   update_bundle_snapshot:
     name: Update embed html bundle snapshot by label
     # This job will only run if the comment was on a pull requests and matches the label
-    if: ${{ github.event.label.name == 'ui change' || contains( github.event.pull_request.labels.*.name, 'ui change') }}
+    # if: ${{ github.event.label.name == 'ui change' || contains( github.event.pull_request.labels.*.name, 'ui change') || github.event == 'SOMETHING???' }}
```

Comment:
being tested in the PR as I can't trigger this locally, most changes in this file will be reverted except this if condition which will be fixed